src/main/terrier.cpp

```diff
@@ -24,8 +25,10 @@ int main() {
     terrier::storage::init_index_logger();
     terrier::storage::init_storage_logger();
     terrier::transaction::init_transaction_logger();
+    terrier::network::TerrierServer terrier_server;
```

Comment:
I find this confusing to have the declaration in the middle of the logger initialization calls.
Move this to after all the logger initialization calls have been made, and possibly separate from them with a blank line.